ConnectorRevit/ConnectorRevit/UI/ConnectorBindingsRevit.Selection.cs

```diff
@@ -243,38 +244,37 @@ private List<Element> GetSelectionFilterObjects(ISelectionFilter filter)
             }
             return selection;
           case "view":
-            var viewFilter = filter as ListSelectionFilter;
+            {
+              var viewFilter = filter as ListSelectionFilter;
+              using var collector = new FilteredElementCollector(currentDoc);
+              var views = collector
+                .WhereElementIsNotElementType()
+                .OfClass(typeof(View))
+                .Where(x => viewFilter.Selection.Contains(x.Name))
+                .ToList();
 
-            var views = new FilteredElementCollector(currentDoc)
-              .WhereElementIsNotElementType()
-              .OfClass(typeof(View))
-              .Where(x => viewFilter.Selection.Contains(x.Name));
+              if (views.Count == 1)
+              {
+                converter.SetContextDocument(views[0]);
```

Comment:
Could you please just add a note here that we're using this to crop the objects according to the view?